### DISCUSSION TITLE
[求助]昨天学校的考试题目

### DISCUSSION THREAD

**Discussion by Zhenjun3** [share121/inter-knot#2995] [其他]
绳网有没有会做的绳匠大佬
（很长很抽象的题目我记不住了）

这是一道6分题，已知矩形ABCD，CD=2，动点E、G，在运动过程中，∠GEF始终等于90度，EG=6，EF=2，∠AFE＞15°，当点G到BD的距离为3+√3时，求∠AFE的度数（cos15°＝（√6+√2）/4＝0.9659）
![IMG_20240818_194943](https://github.com/user-attachments/assets/1d0106bd-385f-42a0-9904-57cdf2cff978)
这是一道3分题，已知EF与圆相切，正方形ABCD，AB、BC边与圆相切，圆的半径为√3，BF=1+√3，求CF
![IMG_20240818_201208.jpg](https://github.com/user-attachments/assets/715439ee-3224-4d66-99b7-e4acf117b191)

**Comment by 514flowey** [comment]
看上去T1明显缺条件，T2图不对。
建议先搞明白题目再来问吧。

**Reply by Zhenjun3** [reply]
抱歉，题目改过来了

**Comment by lllliufeng** [comment]
额，F点怎么说？

**Comment by RaphaelHider** [comment]
T1的条件不充分，画一条线距离BD3+√3，FE与该直线交于H，则E可以在FH上任取

**Reply by Zhenjun3** [reply]
补充：EG=6，EF=2

**Comment by RaphaelHider** [comment]
![Uploading 微信图片_20240819160323.jpg…]()
第二问没说问啥呀

**Reply by Zhenjun3** [reply]
我这小马虎😶，求CF

**Comment by 514flowey** [comment]
第一题 $GF=\sqrt{EG^2+EF^2}=\sqrt{40}$ 定长，相当于 $G$ 在以 $F$ 为圆心，半径 $GF$ 的圆上转。然后  $\sin\angle GFC$ 和 $\sin\angle EFG$ 都很好求。但问题是怎么算都算不出来整数，大概算出来45.97和82.84这样的数值
第二题哪个条件都不会限制正方形的大小，这题目还是有问题啊。
警惕无良委托人，从我做起。

**Comment by Alsuhold** [comment]
这不明示你建系了吗<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by DrLiSi** [comment]
这个委托，我！...我接不了。

**Comment by mmango-bit** [comment]
第一个你以F点为坐标建系，FG长度可以表示，G到BD距离确定，你就可以写出G点的坐标，然后设E点（x，y），EF跟EG距离知道，两距离方程应该就能解出来了

**Comment by kira-1-kira** [comment]
这个我知道，你图应该没截全，右边还有一个电脑，第一个题目名字是叫布什，割门<img src="https://bbs-static.miyoushe.com/static/2024/06/21/afa1dab5a96d2e2bf4ea8c286ae605a8_9163436134828233199.png" alt="陆景和-诶嘿" width="75" >

**Reply by Zhenjun3** [reply]
哈哈哈，你别说，还挺像
